pkg/server/container_status.go

```diff
@@ -100,7 +100,10 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 }
 
 type containerInfo struct {
+	// TODO(random-liu): Add sandboxID in CRI container status.
 	SandboxID   string                   `json:"sandboxID"`
+	Path        string                   `json:"path"`
```

Comment:
These are useful but already in the runtime spec, in the expected location.

Reply:
I'm a little bit unsure about this. Runtime spec is too big, but it is a big of duplication to add it here.

Reply:
Removed.